scripts/setup_for_wrf.py

```diff
@@ -16,102 +16,59 @@
-                        config[k] = config[k].replace(key,value)
-    else:
-        break
-    ##
-    iterationCount += 1
+    ## check that the dates are in the right order
+    assert end_date > start_date, "End date should be after start date"
+except Exception as e:
+    print("Problem parsing start/end times")
+    raise e
 
-## parameters that should agree for the WRF and WPS namelists
-namelistParamsThatShouldAgree = [
-    {'wrf_var': 'max_dom','wrf_group': 'domains', 'wps_var': 'max_dom','wps_group': 'share'},
-    {'wrf_var': 'interval_seconds','wrf_group': 'time_control', 'wps_var': 'interval_seconds','wps_group': 'share'},
-    {'wrf_var': 'parent_id','wrf_group': 'domains', 'wps_var': 'parent_id','wps_group': 'geogrid'}, 
-    {'wrf_var': 'parent_grid_ratio','wrf_group': 'domains', 'wps_var': 'parent_grid_ratio','wps_group': 'geogrid'}, 
-    {'wrf_var': 'i_parent_start','wrf_group': 'domains', 'wps_var': 'i_parent_start','wps_group': 'geogrid'}, 
-    {'wrf_var': 'j_parent_start','wrf_group': 'domains', 'wps_var': 'j_parent_start','wps_group': 'geogrid'}, 
-    {'wrf_var': 'e_we','wrf_group': 'domains', 'wps_var': 'e_we','wps_group': 'geogrid'}, 
-    {'wrf_var': 'e_sn','wrf_group': 'domains', 'wps_var': 'e_sn','wps_group': 'geogrid'}, 
-    {'wrf_var': 'dx','wrf_group': 'domains', 'wps_var': 'dx','wps_group': 'geogrid'}, 
-    {'wrf_var': 'dy','wrf_group': 'domains', 'wps_var': 'dy','wps_group': 'geogrid'}]
 
-assert iterationCount < 10, "Config key substitution exceeded iteration limit..."
+# Perform some checks
+# Iteration count for filling variables
+# Next line moved to load_wrf_config()
+# assert iterationCount < 10, "Config key substitution exceeded iteration limit..."
 
-## check that requisite keys are present
+# check that requisite keys are present
 requisite_keys = ["run_name","start_date", "end_date"]
 for requisite_key in requisite_keys:
-    assert requisite_key in avail_keys, "Key {} was not in the available configuration keys".format(requisite_key)
-
-## parse boolean keys
-truevals = ['true', '1', 't', 'y', 'yes']
-falsevals = ['false', '0', 'f', 'n', 'no']
-boolvals = truevals + falsevals
-bool_keys = ["run_as_one_job", "submit_wrf_now", "submit_wps_component", "only_edit_namelists","restart",'delete_metem_files',"use_high_res_sst_data", 'regional_subset_of_grib_data']
-for bool_key in bool_keys:
-    assert config[bool_key].lower() in boolvals,'Key {} not a recognised boolean value'.format(bool_key)
-    config[bool_key] = (config[bool_key].lower() in truevals)
+    assert requisite_key in config.keys(), "Key {} was not in the available configuration keys".format(requisite_key)
 
+# analysis source
 assert config['analysis_source'] in ['ERAI', 'FNL'], 'Key analysis_source must be one of ERAI or FNL'
 
-# execfile("/opt/Modules/default/init/python")
 
+
+# execfile("/opt/Modules/default/init/python")
 ## make the stack size unlimited (the equivalent of `ulimit -s unlimited` in bash)
```

Comment:
Just remove all the lines related to resource limits. Peter said that they aren't needed anymore